Common/KeyMap.cpp

```diff
@@ -318,9 +323,9 @@ void UpdateConfirmCancelKeys() {
 			confirmKeys.push_back(hardcodedConfirmKeys[i]);
 	}
 
-	const keycode_t hardcodedCancelKeys[] = {
-		NKCODE_ESCAPE,
-		NKCODE_BACK,
+	const KeyDef hardcodedCancelKeys[] = {
+		KeyDef(DEVICE_ID_KEYBOARD, NKCODE_ESCAPE),
+		KeyDef(DEVICE_ID_KEYBOARD, NKCODE_BACK),
```

Comment:
Keyboards don't really have an NK_BACK button so DEVICE_ID_ANY makes more sense here indeed. Unknown is correct about the intention.

So please add support for DEVICE_ID_ANY.